packages/contracts/package.json

```diff
@@ -47,9 +47,16 @@
   },
   "dependencies": {
     "@eth-optimism/core-utils": "0.7.0",
+    "@eth-optimism/smock": "1.1.10",
+    "@ethersproject/abi": "^5",
     "@ethersproject/abstract-provider": "^5.4.1",
-    "@ethersproject/abstract-signer": "^5.4.1",
-    "@ethersproject/hardware-wallets": "^5.4.0"
+    "@ethersproject/abstract-signer": "^5",
```

Comment:
Please follow the same kind of semver used in versioning for the other packages in the `package.json`